src/Symfony/Component/Notifier/Bridge/Mattermost/MattermostTransport.php

```diff
@@ -32,8 +32,7 @@ public function __construct(
         private ?string $path = null,
         ?HttpClientInterface $client = null,
         ?EventDispatcherInterface $dispatcher = null,
-)
-    {
+) {
```

Comment:
missing indentation (same in a few others)

Reply:
I think it's already better than it was. if we want to introduce new feature for Fixer to also indent missing indentation here, can we raise it as new feature request for Fixer and not treat as blocker here ?